pkg/parser/parser.go

```diff
@@ -183,6 +186,7 @@ func (p *Parser) validateActionStatement(stmt *ast.ActionStatement) error {
 			}
 		}
 
+		// if we got here, then we found at least one match
```

Comment:
Not sure this is true since I don't see Verb.Value getting checked for error, but I'll go with it